tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "moduleResolution": "node",
     "allowSyntheticDefaultImports": true,
     "noImplicitThis": true,
-    "noEmitOnError": false,
+    "noEmitOnError": true,
```

Comment:
This will prevent future PRs from being accepted while they have TypeScript compilation errors.